src/unittest/python/cli_tests.py

```diff
@@ -254,3 +315,7 @@ def test_pyb_error_message(self):
             raise PyBuilderException("test")
         except Exception:
             self.assertRegexpMatches(get_failure_message(), r"test \(cli_tests.py\:\d+\)")
+
```

Comment:
We don't generally put `ut.main` in the unit tests.

Reply:
Fixed.